src/CLI/Console.php

```diff
@@ -209,12 +209,12 @@ static public function loop(callable $callback, $sleep = 1 /* 1 second */, calla
                 }
             }
 
-            sleep($sleep);
+            usleep($sleep * 1000000);
```

Comment:
<img width="1185" alt="Screenshot 2022-04-24 at 2 45 11 PM" src="https://user-images.githubusercontent.com/20852629/164974828-8475e352-420b-4590-97ac-21ae6156e7c3.png">

I think we should replace the `sleep` back since `usleep()` cannot guarantee sleeping longer than 1 second